storage/innobase/log/log0recv.cc

```diff
@@ -29,6 +29,10 @@ Created 9/20/1997 Heikki Tuuri
 #include <stdio.h>                              // Solaris/x86 header file bug
 
 #include <vector>
+#ifdef HAVE_SYSTEMD
+#include <systemd/sd-daemon.h>
+#endif
```

Comment:
hm, perhaps this belons to some include file?
together with empty definition for sd_notifyf